tests/helpers.py

```diff
@@ -105,7 +105,7 @@ def read_data(socket, packet_size):
     while len(data) < packet_size:
         next_data = read_socket(socket, packet_size - len(data))
         if not next_data:
-            assert False, "Failed trying to read in the dataz"
+            assert False, "Failed trying to read in the data"
```

Comment:
Rather than a conditional followed by `assert False`, can't this just read `assert next_data, "Failed trying to read in the data"`?